protocol/common/rpcconsumerlogs_test.go

```diff
@@ -0,0 +1,94 @@
+package common
```

Comment:
need to add these to protocol.yaml

Reply:
Fixed 560efde2ef47fcaab162a1d8e14c032f62fe2c44